server/data/nomisUserRolesApi.ts

```diff
@@ -0,0 +1,35 @@
+import config from '../config'
+import RestClient from './restClient'
+
+interface Caseload {
+  id: string
+  name: string
+}
+
+interface UserCaseload {
```

Comment:
This is a minor thing but maybe the name should be plural, e.g. `UserCaseloads`, so that it's more explicit this is returning multiple caseloads.

Reply:
Ok will do a future commit @xoen